### PR TITLE
Model page refactor + counties infrastructure in place for FE

### PR DESCRIPTION
**Pull request by Fiskell**

### CODE REVIEW

File: src/components/CountyMap/CountyMap.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  const state = STATE_CENTERS[location];
+  const counties = require(`./data/${location}.json`);
+
+  useEffect(() => {
+    // https://www.bls.gov/lau/
+    csv('/countyConfigData/sampleData.csv').then(counties => {
+      setData(counties);
+    });
+
+    csv('/countyConfigData/sampleData.csv').then(counties => {
```
Comment (igorkofman) [inline src/components/CountyMap/CountyMap.js]: this seems duplicated?

File: src/components/CountyMap/CountyMap.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      setData(counties);
+    });
+  }, []);
+
+  const colorScale = scaleQuantile()
+    .domain(data.map(d => d.unemployment_rate))
```
Comment (igorkofman) [inline src/components/CountyMap/CountyMap.js]: unemployment rate?

Comment (Fiskell) [reply src/components/CountyMap/CountyMap.js]: This is placeholder data for the cloreopath map of counties.

File: src/screens/ModelPage/ModelPage.js [hunk 3]
```diff
@@ -1,368 +1,172 @@
...
+  const intervention = STATE_TO_INTERVENTION[location];
+  const showModel = !countyView || (countyView && county);
+
+  useEffect(() => {
+    const datasForView = countyView
+      ? modelDatasMap.county
+      : modelDatasMap.state;
+    setModelDatas(datasForView);
+    setInterventions(buildInterventionMap(datasForView));
+  }, [modelDatasMap, countyView]);
```
Comment (adpirz) [inline src/screens/ModelPage/ModelPage.js]: Why is `modelDatasMap` a `useEffect` dependency here?

**Pull request merged**